store/rootmulti/store.go

```diff
@@ -773,8 +777,14 @@ func (rs *Store) Snapshot(height uint64, protoWriter protoio.Writer) error {
 	// and the following messages contain a SnapshotNode (i.e. an ExportNode). Store changes
 	// are demarcated by new SnapshotStore items.
 	for _, store := range stores {
+		rs.logger.Debug("starting snapshot", "store", store.name, "height", height)
 		exporter, err := store.Export(int64(height))
+		if exporter == nil {
+			rs.logger.Error("snapshot failed; exporter is nil", "store", store.name)
```

Comment:
Looking at `storeExport` this will always trigger when there is an error, and the error check below is now not so useful. Was it intended?

Reply:
we modified the exporter function after the fact and probably never cleaned this up